### PR TITLE
Position details

### LINKED ISSUE #21: Position details
- Position id
- Collateral token
- Partition

### PR DESCRIPTION
**Pull request by hpmaxi**
Closes #21

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr37--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr37--conditionaltokensexplorer.review.gnosisdev.com

File: src/util/tools.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const formatBigNumber = (value: BigNumber, decimals: number, precision =
   Number(formatUnits(value, decimals)).toFixed(precision)
 
 export const isConditionIdValid = (conditionId: string): boolean => BYTES_REGEX.test(conditionId)
+export const isBytes32String = (s: string): boolean => BYTES_REGEX.test(s)
```
Comment (mariano-aguero) [inline src/util/tools.ts:39]: Maybe you can use `isBytes32String` in the function `isConditionIdValid`

File: src/pages/positions-list/index.tsx [hunk 2]
```diff
@@ -60,6 +67,7 @@ export const PositionsList = () => {
           }}
           columns={tableColumns}
           data={data || []}
+          onRowClicked={handleRowClick}
```
Comment (mariano-aguero) [inline src/pages/positions-list/index.tsx:70]: Can you add the prop `customStyles` with this values ```const customStyles = {
  rows: {
    style: {
      cursor: 'pointer',
    },
  },
}```  ? , so the user see the row is clickable

File: src/pages/position-detail/PositionDetailWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  const { positionId } = props
+
+  const { data, error, loading } = useQuery<GetPosition>(GetPositionQuery, {
+    variables: { id: positionId },
+  })
+
+  if (data && data.position) {
+    return <PositionDetailItem position={data.position} />
+  } else {
+    return loading ? <div>Loading...</div> : error ? <div>Error...</div> : <div>Not found ...</div>
```
Comment (mariano-aguero) [inline src/pages/position-detail/PositionDetailWrapper.tsx:22]: Regarding the `Error...` string, I think we should display the error from theGraph, like `Error: ${error}` or display the error with  `logger.error(error)`

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr37--conditionaltokensexplorer.review.gnosisdev.com

**Pull request merged**